src/foam/nanos/notification/email/EmailTemplateEngine.js

```diff
@@ -376,7 +376,7 @@ foam.CLASS({
       ps.setString(body);
       ParserContext parserX = new ParserContextImpl();
       parserX.set("sb", sbJoin);
-      parserX.set("x", x);
+      parserX.set("x", getX());
```

Comment:
I dont think this is right

Reply:
@penzital  updated DAOResourceLoader as we discussed